schema/canonical/crc.go

```diff
@@ -3,38 +3,14 @@ package canonical
 import (
 	"bytes"
 	"encoding/binary"
-	"encoding/hex"
 )
 
-// IMplementation of Avro's 64-bit Rabin code for fingerprinting schemas
-
-/*
-long fingerprint64(byte[] buf) {
-  if (FP_TABLE == null) initFPTable();
-  long fp = EMPTY;
-  for (int i = 0; i < buf.length; i++)
-    fp = (fp >>> 8) ^ FP_TABLE[(int)(fp ^ buf[i]) & 0xff];
-  return fp;
-}
-
-static long EMPTY = 0xc15d213aa4d7a795L;
-static long[] FP_TABLE = null;
-
-void initFPTable() {
-  FP_TABLE = new long[256];
-  for (int i = 0; i < 256; i++) {
-    long fp = i;
-    for (int j = 0; j < 8; j++)
-      fp = (fp >>> 1) ^ (EMPTY & -(fp & 1L));
-    FP_TABLE[i] = fp;
-  }
-}
-*/
-
 var FP_TABLE []uint64
 
 const EMPTY uint64 = 0xc15d213aa4d7a795
 
+var HeaderV1 = []byte{0xC3, 0x01}
```

Comment:
This should be in the soe package since it's part of the framing and not part of the CRC calculation.